src/main/java/uk/gov/hmcts/reform/draftstore/data/DraftStoreDAO.java

```diff
@@ -125,12 +125,19 @@ public void update(int id, UpdateDraft draft) {
         );
     }
 
-    public List<Draft> readAll(String userId, String service) {
+    public List<Draft> readAll(String userId, String service, Integer after, int limit) {
         return jdbcTemplate.query(
-            "SELECT * FROM draft_document WHERE user_id = :userId AND service = :service",
+            "SELECT * FROM draft_document "
+                + "WHERE user_id = :userId "
+                + "AND service = :service "
+                + Optional.ofNullable(after).map(a -> "AND id > :after ").orElse("")
```

Comment:
Why pagination is based on primary key values instead of offset?

Reply:
check link from pull request description